### PR TITLE
Ramone/dao linking

### PR DESCRIPTION
**Pull request by Ramone-HL**
Added images and updated Linking Address file

### CODE REVIEW

**Comment by Ramone-HL** [comment]
Update Voting Process>Linking Mainchain address and EON ZEN address

File: docs/governance/03-voting_process/01-linking_mainchain_address.md [hunk 1]
```diff
@@ -4,4 +4,156 @@ title: Linking Mainchain and EON $ZEN Addresses
...
\ No newline at end of file
+## Overview
+
+To enable voting power to all accounts within the Horizen ecosystem, the **Governance Linking** app provides a mechanism to link multiple mainchain addresses to one  EON address. Upon linking, a user’s voting power for the Horizen DAO will be the sum total of their ZEN balance on EON, plus the ZEN balance of all of their linked mainchain addresses. This makes it easier for users to vote with the totality of their ZEN without having to move assets. 
+
+### Configuring a Network
+
+Before linking an EON wallet, make sure that the appropriate network is available. If not, add the network to the EON wallet and set it as the primary network. 
+
+See [Connect your Wallet](https://docs.horizen.io/horizen_eon/connect/connect_your_wallet) in the EON documentation.
```
Comment (rabyyuson) [inline docs/governance/03-voting_process/01-linking_mainchain_address.md:15]: let's make this a relative link:

```suggestion
See [Connect your Wallet](/horizen_eon/connect/connect_your_wallet) in the EON documentation.
```

File: docs/governance/03-voting_process/01-linking_mainchain_address.md [hunk 2]
```diff
@@ -4,4 +4,156 @@ title: Linking Mainchain and EON $ZEN Addresses
...
+
+## Linking Addresses 
+
+Perform the following steps to link one EON address to one or more mainchain addresses.
+
+**Note:** MetaMask is used for providing the EON address.
+
+
+
+1. Open the **[Horizen Governance ](https://development.evm-backward-transfer-ui-e76.pages.dev/governance/dao)**page.
```
Comment (rabyyuson) [inline docs/governance/03-voting_process/01-linking_mainchain_address.md:26]: @Ramone-HL currently this links to a dev/preview link. Also, its [production page](https://eon.horizen.io/governance/dao) counterpart  is hidden/not accessible through the sidebar.

@yurikoinaba are we making the [Governance page](https://eon.horizen.io/governance/dao) public with this change? I also noticed that most of the buttons like the "Mainchain Link Guide", "Snapshot", and "Discourse" don't do anything when clicked and will require an update.

File: docs/governance/03-voting_process/01-linking_mainchain_address.md [hunk 3]
```diff
@@ -4,4 +4,156 @@ title: Linking Mainchain and EON $ZEN Addresses
...
+
+12. In the **Horizen Governance Linked Addresses** pane, paste the signature in the **Mainchain Signature** field.  
+
+<p>
+<img src={require("/img/docs/dao/dao_gov_addresslink1.png").default} alt="Link Addresses" width="300" height="200" />
+</p>
+
+
+
+13. Click **Link**. The Pending pop-up appears stating that the transaction has been submitted.
```
Comment (rabyyuson) [inline docs/governance/03-voting_process/01-linking_mainchain_address.md:111]:
```suggestion
13. Click **Link**. The Pending pop-up appears stating that the transaction is being submitted.
```

File: docs/governance/03-voting_process/01-linking_mainchain_address.md (hunk 2 above)
Comment (Ramone-HL) [reply docs/governance/03-voting_process/01-linking_mainchain_address.md:26]: The production link will be: https://eon.horizen.io/governance/dao and is not available.

Comment (Ramone-HL) [reply docs/governance/03-voting_process/01-linking_mainchain_address.md:26]: For Discourse the link is:  http://forum.horizen.io/

**Pull request closed without merging**